OpenRA.Game/World.cs

```diff
@@ -169,6 +169,8 @@ public bool ToggleInputMode<T>() where T : IOrderGenerator, new()
 			}
 		}
 
+		public readonly MusicPlaylist Playlist;
```

Comment:
This is only accessed from two places (`MediaGlobal` and `MusicPlayerLogic`).  It would be better if you queried the trait in those two places and removed this reference.